src/XMakeBuildEngine/Evaluation/ProjectRootElementCache.cs

```diff
@@ -194,7 +194,7 @@ internal ProjectRootElementCache(bool autoReloadFromDisk)
         /// <param name="preserveFormatting"><code>true</code> to the project was loaded with the formated preserved, otherwise <code>false</code>.</param>
         /// <returns>The ProjectRootElement instance if one exists.  Null otherwise.</returns>
         internal ProjectRootElement Get(string projectFile, OpenProjectRootElement openProjectRootElement, bool isExplicitlyLoaded,
-            bool preserveFormatting)
+            bool? preserveFormatting)
```

Comment:
Change `ProjectRootElement.Open` and `ProjectRootElement.TryOpen` to pass `preserveFormatting: null` here when the overload without `preserveFormatting` is used.